src/EdFi.AnalyticsMiddleTier.DataStandard32/Asmt/PostgreSQL/0016-View-StudentAssessmentFact-Alter.sql

```diff
@@ -0,0 +1,132 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+DROP VIEW IF EXISTS analytics.asmt_StudentAssessmentFact;
+
+CREATE OR REPLACE VIEW analytics.asmt_StudentAssessmentFact 
+AS
+	SELECT 
+           CONCAT(StudentAssessment.AssessmentIdentifier, '-', StudentAssessment.Namespace, '-', StudentAssessment.StudentAssessmentIdentifier, '-', StudentAssessment.StudentUSI) AS StudentAssessmentKey,
+           CASE WHEN SASOA.StudentUSI IS NULL
+               THEN ''
+               ELSE CONCAT(SASOA.StudentUSI, '-', SASOA.IdentificationCode, '-', SASOA.AssessmentIdentifier, '-', SASOA.StudentAssessmentIdentifier, '-', SASOA.Namespace)
+           END AS StudentObjectiveAssessmentKey,
+           CASE WHEN SASOA.AssessmentIdentifier IS NULL
+               THEN ''
+               ELSE CONCAT(SASOA.AssessmentIdentifier, '-', SASOA.IdentificationCode, '-', SASOA.Namespace)
+           END AS ObjectiveAssessmentKey, 
+           CONCAT(Assessment.AssessmentIdentifier, '-', StudentAssessment.Namespace) AS AssessmentKey, 
+           Assessment.AssessmentIdentifier, 
+           StudentAssessment.Namespace, 
+           StudentAssessment.StudentAssessmentIdentifier, 
+           StudentAssessment.StudentUSI, 
+           CONCAT(Student.StudentUniqueId, '-', StudentSchoolAssociation.SchoolId) AS StudentSchoolKey, 
+           School.SchoolId AS SchoolKey, 
+           CONVERT(VARCHAR, StudentAssessment.AdministrationDate, 112) AS AdministrationDate, 
```

Comment:
I think there are some pending changes in this postgres version. For example, this line doesn't work in postgres.